test-data/unit/check-incremental.test

```diff
@@ -2195,9 +2196,9 @@ from b import x
 1 + 1
 [out]
 [out2]
-tmp/b.py:2: error: Invalid type "c.C"
+tmp/b.py:2: error: Invalid type "c.C": Cannot use function as type, use Callable[...] or a protocol instead
```

Comment:
Again, the first part could be shortened to `Function "c.C" is not valid as a type`. I'd move the final part to a separate note and rephrase it, since it's not clear of `Callable[...]` or a protocol is actually what the user wants. Maybe something like this:

```
b.py:2: error: Function "c.C" is not a valid type
b.py:2: note: Perhaps you need Callable[...] or a callback protocol?
```